micromax/b-micromax-md3-pc/exporter.py

```diff
@@ -518,7 +524,8 @@ def _do_start_raster_scan(
         return self._add_task("Start RASTER SCAN", 5.2)
 
     async def _move_motors_simultaneously(
-        self, motors: list[MovedMotor], move_duration: float
+        self, motors: list[MovedMotor],
+        move_duration: float
```

Comment:
Strange formatting here:
```suggestion
        self,
        motors: list[MovedMotor],
        move_duration: float,
```
or maybe leave as it was before, I can not see any necessary change here.